src/app/widgets/inlineinput/inlineinput.component.less

```diff
@@ -1,8 +1,8 @@
 @import (reference) './../../../assets/stylesheets/_base';
 
-.inlineinput-container {
+.f8-inlineinput {
```

Comment:
Missing `planner` domain in selector.

Reply:
inline input is from common widgets which can be contributed outside planner. Should we nest these under planner domain?